libbeat/tests/system/requirements_aix.txt

```diff
@@ -0,0 +1,49 @@
+attrs==19.3.0
+autopep8==1.5.4
+backports.ssl-match-hostname==3.5.0.1
+cached-property==1.4.2
+certifi==2018.1.18
+chardet==3.0.4
+deepdiff==4.2.0
+docker==4.1.0
+docker-pycreds==0.4.0
+dockerpty==0.4.1
+docopt==0.6.2
+elasticsearch==7.8.1
+enum34==1.1.6
+idna==2.6
+importlib-metadata==1.7.0
+iniconfig==1.0.1
+ipaddress==1.0.19
+Jinja2==2.11.3
+jsondiff==1.1.2
+jsonschema==3.2.0
+kafka-python==1.4.3
+MarkupSafe==1.1.1
+more-itertools==8.4.0
+ordered-set==3.1.1
+packaging==20.4
+parameterized==0.7.0
+pluggy==0.13.1
+py==1.10.0
+pycodestyle==2.6.0
+pyparsing==2.4.7
+pyrsistent==0.16.0
+pytest==6.2.4
+pytest-otel>=0.0.4
+pytest-rerunfailures==9.1.1
+pytest-timeout==1.4.2
+PyYAML==5.4.1
+redis==2.10.6
+requests==2.25.1
+semver==2.8.1
+setuptools==47.3.2
+six==1.15.0
+stomp.py==4.1.22
+termcolor==1.1.0
+texttable==0.9.1
+toml==0.10.1
+urllib3==1.26.5
+wcwidth==0.2.5
+websocket-client==0.47.0
+zipp>=1.2.0,<=3.1.0
```

Comment:
@kuisathaverat could you confirm that these new dependencies are expected to be added? The original PR does not have so many new dependencies added, so it looks a bit suspicious.

Reply:
This is a new file, and https://github.com/elastic/beats/pull/28826 was not backported to 7.17,

I'll delete it then